node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -348,12 +348,14 @@ class NodeRuntime private[node] (
     multiParentCasperRef <- MultiParentCasperRef.of[Effect]
     lab                  <- LastApprovedBlock.of[Task].toEffect
     labEff               = LastApprovedBlock.eitherTLastApprovedBlock[CommError, Task](Monad[Task], lab)
+    commTmpFolder        = conf.server.dataDir.resolve("tmp").resolve("comm")
+    _                    <- Task.delay(commTmpFolder.toFile.delete()).attempt.toEffect
```

Comment:
logs? if you can't remove it, folder will grow. add `warn`?

Reply:
Ok, then we need some more sophisticated logic that checks if the folder exists. If the folder exists then try to delete. Log in case of an exception. Log in case of result `false`. We have already this logic in Tcp TL. Maybe I can move it to `shared`.

Reply:
Mabye add a log message saying 'deleting comm tmp folder $folderPath'?